code/components/citizen-scripting-v8/src/V8ScriptRuntime.cpp

```diff
@@ -7,6 +7,15 @@
 
 #include "StdInc.h"
 #include "fxScripting.h"
+#include <ManifestVersion.h>
+
+#ifndef IS_FXSERVER
+static constexpr std::pair<const char*, ManifestVersion> g_scriptVersionPairs[] = {

```

Comment:
What is there to remain compatible with for JS? It makes more sense to start off with only universal natives, potentially binding them to the null GUID so that it'll work even in older versions.